sources/packages/web/src/views/student/StudentProfile.vue

```diff
@@ -5,6 +5,48 @@
     information needs to be changed please visit
     <a href="http://id.gov.bc.ca" target="_blank">id.gov.bc.ca</a>.
   </Message>
+  <span v-if="showApplyPDButton">
+    <v-btn
+      color="primary"
+      @click="applyPDStatus()"
+      v-if="showApplyPDButton"
+      :disabled="disableBtn"
+    >
+      Apply for PD status
+      <span v-if="disableBtn">
+        &nbsp;&nbsp;
+        <ProgressSpinner style="width:30px;height:25px" strokeWidth="10"
+      /></span>
+    </v-btn>
+  </span>
+  <span v-else>
+    <Message
```

Comment:
Should we add a message for when the SIN is not validated yet and the student must wait for this process to happen?

Reply:
We will have further requirements on what a student can do when their SIN is not validated. Not required at this time but there will be a requirement for it in the future

Reply:
that's a good point, @abschwenker  do we need to show msg to student if their SIN is not validated yet